README.md

```diff
@@ -1,2 +1,197 @@
-# event-monitoring-service
-Dynamic event monitoring service (DEMS)
+# Event Monitoring Service
+
+<div align="center">
+<img alt="GitHub Actions Workflow Status" src="https://img.shields.io/github/actions/workflow/status/tazama-lf/event-monitoring-service/node.js.yml">
+</div>
+
+## Overview
+
+High-performance middleware for the Tazama FRMS that receives, validates, and routes financial transaction requests to the Event Director.
+
+**Key Capabilities:**
+
+- Validates incoming transactions using JSON Schema
+- Transforms XML/JSON transaction data
+- Routes processed events via NATS messaging
+- Persists transaction relationships in PostgreSQL
+- Provides Redis caching for performance
+
+### Setting Up
+
+```sh
+git clone https://github.com/tazama-lf/event-monitoring-service
+cd event-monitoring-service
+```
+
+You then need to configure your environment: a [sample](.env.template) configuration file has been provided and you may adapt that to your environment. Copy it to `.env` and modify as needed:
+
+```sh
+cp .env.example .env
+```
+
+#### Prerequisites
+
+- Node.js 20+
+- PostgreSQL 15+
+- Redis 7+
+- NATS Server 2.10+
+
+#### Build and Start
+
+```sh
+npm install
+docker-compose up -d redis nats postgres
```

Comment:
I think we also need to think of the bigger picture and highlight what components of the full-stack-docker are reusable (if any). I see we have NATS here which we will probably using to route to the event-director. If so, then that same NATS instance should also be accessible by the event-director. 

I'm more so asking the question - If I deploy full-stack-docker - do I need to deploy anything else to run this service?